src/core/dbus-timer.c

```diff
@@ -137,6 +137,38 @@ const sd_bus_vtable bus_timer_vtable[] = {
         SD_BUS_VTABLE_END
 };
 
+static int timer_add_one_monotonic_spec(
+                Timer *t,
+                const char *name,
+                TimerBase base,
+                UnitWriteFlags flags,
+                usec_t usec,
+                sd_bus_error *error) {
+
+        if (!UNIT_WRITE_FLAGS_NOOP(flags)) {
```

Comment:
I'd avoid the deep indentation here, and simple exit early here if UNIT_WRITE_FLAGS_NOOP holds

Reply:
Yeah, I thought about that, but there's `return 1` at the end, and I prefer to have that once.